app/src/main/res/layout/patientsummary_bloodsugar_item_content.xml

```diff
@@ -21,49 +20,66 @@
     android:layout_marginBottom="@dimen/spacing_4"
     android:src="@drawable/ic_blood_sugar_outline"
     app:layout_constraintBottom_toBottomOf="parent"
-    app:layout_constraintEnd_toStartOf="@id/readingTextView"
-    app:layout_constraintHorizontal_bias="0"
-    app:layout_constraintHorizontal_chainStyle="packed"
+    app:layout_constraintEnd_toStartOf="@id/flow"
     app:layout_constraintStart_toStartOf="parent"
     app:layout_constraintTop_toTopOf="parent"
     tools:ignore="ContentDescription" />
 
-  <org.simple.clinic.widgets.TintableCompoundDrawableTextView
+  <androidx.constraintlayout.helper.widget.Flow
```

Comment:
~Error: Class referenced in the layout file was not found in the project or libraries? I checked we do have the library. Can you take a look?~ Solved in the next commit